stream_alert_cli/lookup_tables/handler.py

```diff
@@ -21,33 +24,149 @@
 
 
 class LookupTablesCommand(CLICommand):
-
     description = 'Describe and manage your LookupTables'
 
     @classmethod
     def setup_subparser(cls, subparser):
-        """Add subparser for LookupTables"""
+        # FIXME (derek.wang) Refactor this into a more robust command-nesting framework
+        subcommands = cls._subcommands()
         set_parser_epilog(
             subparser,
             epilog=(
                 '''\
+Available Sub-Commands:
+
+{}
+
+Examples:
+
+    manage.py lookup-tables [describe-tables|get|set]
+                '''.format('\n'.join([
```

Comment:
do you want these trailing spaces `                ` here ..?

Reply:
I think probably somewhere in my brain I mixed up a string and a heredoc and  ... yea